scripts/network-init

```diff
@@ -108,6 +108,12 @@ rename_network_label() {
     done
 }
 
+maybe-enable-tls-verification() {
```

Comment:
Given that `$XENSOURCE_INVENTORY` is in shell syntax, would not not be easier to `source $XENSOURCE_INVENTORY` and simply look for the value of `FIRSTBOOT_ENABLE_TLS_VERIFICATION`?

Reply:
note: `$XENSOURCE_INVENTORY` is sourced in line 11